### DISCUSSION TITLE
Bypass YouTube API quota

### DISCUSSION THREAD

**Discussion by kiatchun** [ThioJoe/YT-Spammer-Purge#241] [Ideas]
Would it be possible if the config file could point to multiple client_secrets.json from multiple google cloud projects? If the first one's quota is used up, the script will try out the second one. If it works, I google account can theoretically have 1200000 quota units.

**Comment by ThioJoe** [comment]
Hmm, not a bad idea. However, because you'd need to log into each project separately, it wouldn't be able to continue if the quota gets used up in the middle of a scan.

**Reply by kiatchun** [reply]
But you would be able to run the next scan without changing the file(s) manually.

**Reply by ghost** [reply]
Is this still true with the token.pickle files?

**Comment by sakgoyal** [comment]
the application could spin up a 2nd process that uses the 2nd account right? not exactly sure how easy this would be but it sounds possible.

**Comment by Masamune3210** [comment]
LTT covered how to set up multiple users and bypass the API limit in their feature on this
